draft-ietf-rats-corim.md

```diff
@@ -1656,14 +1656,17 @@ A Reference Value consists of an `environment-map` plus a `measurement-map`. In
 Reference Values are represented more compactly by letting one `environment-map`
```

Comment:
This statement is inaccurate. Only the `reference-triple-record` and `coswid-triple-record` contain Reference Values. The coswid triple has multiple objects for a common subject, but it isn't a `measurement-map`.

Reply:
I will change Reference Value to stateful environment when fixing [#80](https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/80)